EIPS/eip-7739.md

```diff
@@ -0,0 +1,207 @@
+---
+eip: 7739
```

Comment:
```suggestion
eip: 7521
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Thank you! Will update right away